### PR TITLE
AOB-1455: Upgrade onboarder installation and troubleshooting doc

### PR DESCRIPTION
**Pull request by PierrickMartos**
**Description**

Upgrade onboarder installation documentation and troubleshooting page since we have renamed the env var for Google PubSub.

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: onboarder/troubleshooting/index.rst [hunk 1]
```diff
@@ -29,3 +29,10 @@ You can check if Google Pub/Sub access is successfully configured by using the f
     :scale: 100 %
     :alt: Result of the check pubsub access command
     :align: center
+
+Proposals from Onboarder don't show up into the PIM
```
Comment (phaseinducer) [inline onboarder/troubleshooting/index.rst:33]:
```suggestion
Proposals from Onboarder don't show up in the PIM
```

File: onboarder/troubleshooting/index.rst [hunk 2]
```diff
@@ -29,3 +29,10 @@ You can check if Google Pub/Sub access is successfully configured by using the f
...
+---------------------------------------------------
+
+If you don't received proposals into the PIM, double check that you have rightfully setup and configure your workers, see :doc:`synchronization </onboarder/synchronization/index>` page.
```
Comment (phaseinducer) [inline onboarder/troubleshooting/index.rst:36]:
```suggestion
If you don't receive proposals in the PIM, double check that you properly set up and configured your workers, see :doc:`synchronization </onboarder/synchronization/index>` page.
```
"rightfully" means "à juste titre"^^

**Review by phaseinducer — APPROVED** [review]

File: onboarder/troubleshooting/index.rst (hunk 2 above)
Comment (PierrickMartos) [reply onboarder/troubleshooting/index.rst]: Heureusement que t'es là pour corriger mon anglais :D

**Review by ValentinMerlet — APPROVED** [review]

**Review by kpAkeneo — APPROVED** [review]

**Review by kpAkeneo — APPROVED** [review]

File: onboarder/installation/index.rst [hunk 3]
```diff
@@ -102,31 +102,31 @@ Here are two examples in order to define environment variables:
...
 
 **Optional variables**
 
-+----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| PUBSUB_EMULATOR_HOST | Use this if you want to use a Pub/Sub emulator during development. In this case you don't need the serviceAccount.json file neither set the ONBOARDER_GOOGLE_APPLICATION_CREDENTIALS variable. |
-+----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
++----------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| PUBSUB_EMULATOR_HOST | Use this if you want to use a Pub/Sub emulator during development. In this case you don't need the onboarderServiceAccount.json file neither set the ONBOARDER_GOOGLE_APPLICATION_CREDENTIALS variable. |
```
Comment (ValentinMerlet) [inline onboarder/installation/index.rst:128]: In this case you don't need the ``onboarderServiceAccount.json``

**Review by ValentinMerlet — APPROVED** [review]

File: onboarder/installation/index.rst (hunk 3 above)
Comment (PierrickMartos) [reply onboarder/installation/index.rst:128]: Thanks but I don't understand what you propose, to remove words after `onboarderServiceAccount.json` ?

**Pull request merged**